### PR TITLE
solution

### PR DESCRIPTION
**Pull request by milenadiamonds1**
[DEMO LINK ](https://milenadiamonds1.github.io/layout_product-cards/)
[TEST REPORT](https://milenadiamonds1.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (vladyslav-mosor) [inline src/index.html:6]: Please fix indentations for attributes. And string that has length over 80 should be wrapped

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+      src="/images/imac.jpeg" 
+      alt="Apple A1419 iMac 27 Retina 5K" 
+      width="160" 
+      height="134"
+      class="card__image" 
+      >
```
Comment (vladyslav-mosor) [inline src/index.html:18]: Please fix indentations

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,56 @@
...
+      <div class="card__block card__block--second">
+        <p class="card__price">Price:</p>
+
+        <p class="card__price-value">$2,199</p>
+      </div>
+      <a 
+      href="#" 
+      class="card__link"
+      data-qa="hover"
+      >
```
Comment (vladyslav-mosor) [inline src/index.html:51]: Please fix indentations

File: src/styles/blocks/card.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+  
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
```
Comment (vladyslav-mosor) [inline src/styles/blocks/card.css:67]: Don`t set height prop for link. Let the content dictate it

File: src/styles/blocks/card.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  text-transform: uppercase;
+  font-family: "Roboto-Bold", sans-serif;
+  vertical-align: middle;
+  color: var(--white);
+}
+  
+.card__link:hover {
+  background: var(--white);
+  color: var(--specificBlue);
+  border: var(--specificBlue);
```
Comment (vladyslav-mosor) [inline src/styles/blocks/card.css:82]:
```suggestion
  border-color: var(--specificBlue);
```

**Review by vladyslav-mosor — CHANGES_REQUESTED** [review]

**Comment by vladyslav-mosor** [comment]
Your link doesn`t have border on hover state as it shows on mockup.

File: src/index.html [hunk 6]
```diff
@@ -3,18 +3,25 @@
...
     <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="
+          width=device-width, 
+          user-scalable=no, 
+          initial-scale=1.0, 
+          maximum-scale=1.0, 
+          minimum-scale=1.0
+        "
+    >
```
Comment (vladyslav-mosor) [inline src/index.html:13]: image

Fix like it shows at the example

**Review by vladyslav-mosor — APPROVED** [review]

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,59 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width, 
+            user-scalable=no, 
+            initial-scale=1.0, 
+            maximum-scale=1.0, 
+            minimum-scale=1.0
+          "
+    >
```
Comment (Andr1yK) [inline src/index.html:13]:
```suggestion
    <meta
    	name="viewport"
      content="
        width=device-width, 
        user-scalable=no, 
        initial-scale=1.0, 
        maximum-scale=1.0, 
        minimum-scale=1.0
      "
    >
```

File: src/styles/variables.css [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+@font-face {
+  font-family: Roboto-Regular;
+  src: url("/Roboto/Roboto-Regular.ttf");
+}
+
+@font-face {
+  font-family: Roboto-Bold;
+  src: url("/Roboto/Roboto-Bold.ttf");
+}
```
Comment (Andr1yK) [inline src/styles/variables.css:22]: image

File: src/styles/reset.css [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  font-weight: normal;
+}
+  
+body {
+  margin: 0;
+}
+  
+p {
+  margin: 0;
+}
```
Comment (Andr1yK) [inline src/styles/reset.css:18]:
```suggestion
body,
p {
  margin: 0;
}
```

**Review by Andr1yK — CHANGES_REQUESTED** [review]
why do u need all font weights?
image

**Review by Andr1yK — CHANGES_REQUESTED** [review]
Never add dist/build folders to PR, let's remove it
image

File: src/styles/variables.css [hunk 10]
```diff
@@ -0,0 +1,25 @@
+:root {
+  --basic: #060b35;
+  --specificGrey: #616070;
+  --specificBlue: #00acdc;
+  --white: #fff;
+  --borderColor: #f3f3f3;
```
Comment (Andr1yK) [inline src/styles/variables.css:6]: what is basic?) is it a color, or maybe a size?

Comment (Andr1yK) [reply src/styles/variables.css:6]: good practice for naming colors is a `--c` + `color name`
F.E: `--c-white`

File: src/styles/blocks/stars.css [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+  
+.stars.stars--1 :first-child,
+.stars.stars--2 :nth-child(-n+2),
+.stars.stars--3 :nth-child(-n+3),
+.stars.stars--4 :nth-child(-n+4),
+.stars.stars--5 .stars__star {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
```
Comment (Andr1yK) [inline src/styles/blocks/stars.css:37]:
```suggestion
}

```

File: src/styles/blocks/stars.css [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+  margin-right: 4px;
+}
+  
+.stars.stars--0 .stars__star,
+.stars.stars--1 :not(:first-child),
+.stars.stars--2 :nth-last-child(-n+3),
+.stars.stars--3 :nth-last-child(-n+2),
+.stars.stars--4 :nth-last-child(-n+1),
+.stars.stars--5 .stars__star {
+  background-image: url("/images/star.svg");
```
Comment (Andr1yK) [inline src/styles/blocks/stars.css:25]: let's avoid too many nth-child, and set background-image as default for stars__star class

File: src/styles/blocks/card.css [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+  text-transform: uppercase;
+  font-family: "Roboto-Bold", sans-serif;
+  color: var(--white);
+}
+  
+.card__link:hover {
+  background: var(--white);
+  color: var(--specificBlue);
+  border-color: var(--specificBlue);
+}
```
Comment (Andr1yK) [inline src/styles/blocks/card.css:81]:
```suggestion
}

```

File: src/index.html [hunk 14]
```diff
@@ -1,13 +1,60 @@
...
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+    	name="viewport"
```
Comment (Andr1yK) [inline src/index.html:6]:
```suggestion
      name="viewport"
```

**Review by Andr1yK — CHANGES_REQUESTED** [review]
Do u use Roboto-Thin file?

**Pull request closed without merging**